src/widgets/tooltip/ProfileTooltip.scss

```diff
@@ -0,0 +1,11 @@
+.ProfileTooltip {
+  font-size: 16px;
+  text-align: center;
+  width: 250px;
+  height: 260px;
```

Comment:
The content of the profile tooltip is not visible entirely, it's crop on the user location, can you make it more height like 280px?